### PR TITLE
fRO tables update

### PR DESCRIPTION
**Pull request by sctnightcore**

### CODE REVIEW

File: tables/fRO/maps.txt [hunk 1]
```diff
@@ -862,24 +862,24 @@ teg_dun02.rsw#Subterranean Guild Dungeon#
...
 gl_chyard_.rsw#Churchyard#
 2@gl_k.rsw#1st floor of Old Glast Heim Chivalry#
 1@gl_k.rsw#2nd floor of Old Glast Heim Chivalry#
 
-//Clocktower_nightmare
+#Clocktower_nightmare
 c_tower2_.rsw#Twisted Time Clock Tower 2nd Floor#
 c_tower3_.rsw#Twisted Time Clock Tower 3rd Floor#
-//뾵뿵샇죧샻
+#¿µ¿õÀÇÈçÀû
```
Comment (Nipodemos) [inline tables/fRO/maps.txt]: this need to be fixed

File: tables/fRO/maps.txt [hunk 2]
```diff
@@ -844,7 +844,7 @@ ecl_tdun04.rsw#Bifrost Floor 4#
 ecl_hub01.rsw#Eclage Perimeter#
 que_avan01.rsw#Avant's Laboratory#
 
-//냸벺샼 TE 몹믧룊
+#°ø¼ºÀü TE º¹»ç¸Ê
```
Comment (Nipodemos) [inline tables/fRO/maps.txt]: this too

**Comment by Nipodemos** [comment]
sorry, but there is a problem with how you encode these files. All korean characters are not displaying correctly, and should be fixed. 
are you sure you encoded as UTF-8 while generating those files?

**Comment by sctnightcore** [comment]
@Nipodemos  yes, all file encode with UTF-8

**Comment by ya4ept** [comment]
Is this query still relevant? I propose to close

**Comment by Nipodemos** [comment]
i think we can close this too.
let just @sctnightcore give his word

**Comment by sctnightcore** [comment]
@Nipodemos  I'm lazy 

close it

**Pull request closed without merging**